alarm/binance_utils.py

```diff
@@ -7,15 +7,42 @@
 logger = logging.getLogger(f'{__name__}')
 
 
-def get_binance_trade_pairs():
+def get_binance_valid_trade_pairs_with_base_quote_assets():
     binance_exchange_info_url = "https://api.binance.com/api/v3/exchangeInfo"
     response = requests.get(binance_exchange_info_url)
+    if response.status_code != 200:
+        raise requests.exceptions.RequestException("API request for get binance exchange data was failed")
 
-    if response.status_code == 200:
-        valid_list_of_coin_names = [symbol['symbol'].lower() for symbol in response.json()['symbols']]
-        return valid_list_of_coin_names
-    else:
-        return []
+    data = response.json()
+    trade_pairs = data['symbols']
+    trade_pairs_with_base_quote_assets = {}
+    for trade_pair in trade_pairs:
+        base_asset = trade_pair['baseAsset']
+        quote_asset = trade_pair['quoteAsset']
+        trade_pairs_with_base_quote_assets[trade_pair['symbol']] = {'baseAsset': base_asset, 'quoteAsset': quote_asset}
+    return trade_pairs_with_base_quote_assets
+
+
+def format_trade_pair_for_message(trade_pair):
+    binance_valid_trade_pairs_with_base_quote_assets = get_binance_valid_trade_pairs_with_base_quote_assets()
+
+    for trade_pair_with_base_quote_asset in binance_valid_trade_pairs_with_base_quote_assets:
+        if trade_pair.upper() in trade_pair_with_base_quote_asset:
+            base_asset = binance_valid_trade_pairs_with_base_quote_assets[trade_pair_with_base_quote_asset]['baseAsset']
```

Comment:
Looks like we can make this directly
base_asset = trade_pair_with_base_quote_asset['baseAsset']

Reply:
++